### ISSUE TITLE
modprobe: FATAL: Module wireguard not found in directory

### ISSUE THREAD

**Issue by drew2a** [drew2a/wireguard#10]
OS: Debian 9
VPS: Digital Ocean

```
> sudo modprobe wireguard
modprobe: FATAL: Module wireguard not found in directory /lib/modules/4.9.0-13-amd64
```

**Comment by drew2a** [comment]
References:
* https://www.ivpn.net/knowledgebase/241/Linux---WireGuard---RTNETLINK-answers-Operation-not-supported.html#:~:text=Linux%20%2D%20WireGuard%20%2D%20%22RTNETLINK%20answers%3A%20Operation%20not%20supported%22,-If%20you%20are&text=A%20chance%20exists%20that%20the,to%20raise%20the%20trust%20level.
* https://ubuntu.com/blog/how-to-sign-things-for-secure-boot
* https://askubuntu.com/questions/1174930/cant-load-wireguard-module
* https://www.linuxbabe.com/debian/wireguard-vpn-server-debian
* https://askubuntu.com/questions/1244072/wireguard-fails-to-build-module-even-tho-kernel-headers-are-installed
* https://www.reddit.com/r/WireGuard/comments/cy8nk0/wireguard_module_not_loading_with_centos_7_kernel/

**Comment by arthuriantech** [comment]
Hi, I had this problem tonight on Ubuntu 18.04. I rebooted VDS after `apt install -y linux-headers-"$(uname -r)"` command and run `./wg-ububtu-server-up.sh` again. It fix the problem.

**Comment by drew2a** [comment]
@arthuriantech nice to see you in this repo :) thank you for your comment.

Actual `./wg-ububtu-server-up.sh` has this command in the execution:

https://github.com/drew2a/wireguard/blob/master/wg-ububtu-server-up.sh#L18

Thanks to gods, ubuntu works just fine.

But I have found a problem with Debian:

If you execute  `wg-debian-server-up.sh` it will set up VPN as expected (and at the end VPN is worked):
```bash
sudo apt install -y linux-headers-"$(uname -r)"
echo "deb http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list
apt-get update -y
sudo apt install -y wireguard
sudo modprobe wireguard
```

But, if you turn on `stop-on-error` flag:

``` bash
set -e
```

Then you will see that these commands run with errors.
One of the errors is `modprobe: FATAL: Module wireguard not found in directory` 

I discovered this last night and I did not have time for an investigation.
I plan to investigate the issue next weekend.
